src/project_generator.rs

```diff
@@ -36,6 +39,18 @@ impl Generator {
             Some(d) => d,
             None => project_title.clone(),
         };
+        
+        if fs::metadata(&directory).map(|metadata| metadata.is_dir()).unwrap_or(false) {
+            println!(
+                "{}",
+                format!(
+                    "{} Directory '{}' already exists. Please use other name for directory.",
+                    "🚫 Error:".red(),
+                    &directory.bold(),
+                )
+            );
+            process::exit(1);
+        }
```

Comment:
Can you make the `Generator::new()` function to return a `Result` so that we can match on the error and shutdown gracefully, instead of `process::exit(1)` ?

Also I'm already using `log::error`. I think it would be good to implement a new custom error and return it for this, see `errors.rs`